frontend/crowdeye/management/commands/startserver.py

```diff
@@ -33,11 +35,12 @@ def get_cams():
 
 
             print('t', DATA_LIST)
+            writeEntry(DATA_LIST[0]["node_id"], DATA_LIST[0]["crossed_left"], DATA_LIST[0]["crossed_right"], DATA_LIST[0]["total_crossed"], DATA_LIST[0]["total_people"])
```

Comment:
Would their be a better way to do this

Reply:
Also need to make it possible to send the entire data list as there may be multiple nodes